world/client.cpp

```diff
@@ -156,7 +156,7 @@ void Client::SendLogServer()
 	if(RuleB(World, IsGMPetitionWindowEnabled))
 		l->enable_petition_wnd = 1;
 
-	if((RuleI(World, FVNoDropFlag) == 1 || RuleI(World, FVNoDropFlag) == 2) && GetAdmin() > RuleI(Character, MinStatusForNoDropExemptions))
+	if(RuleI(World, FVNoDropFlag) == 1 || (RuleI(World, FVNoDropFlag) == 2 && GetAdmin() >= RuleI(Character, MinStatusForNoDropExemptions)))
```

Comment:
Would like to see these numbers turned into constants - magic numbers are not very readable in the source by new eyes much less mine

It could go in `eq_constants.h`